Cargo.toml

```diff
@@ -47,6 +47,7 @@ reqwest-client-rustls = ["opentelemetry-http/reqwest", "reqwest/rustls-tls"]
 
 [dependencies]
 async-trait = "0.1"
+backon = "1.5.1"
```

Comment:
Actually, what kind of sleeper does that use. I suspect tokio. It's probably better to use a custom sleeper that makes use of the opentelemetry runtime

Reply:
https://docs.rs/opentelemetry_sdk/latest/opentelemetry_sdk/runtime/trait.Runtime.html#tymethod.delay would be good. But that's behind a feature. I'm going to check how other crates do that, I think.

Reply:
https://github.com/open-telemetry/opentelemetry-rust/issues/2643#issuecomment-2688739272 suggests delay could be removed. In that case we would have to provide that option in this crate. Hm.